package.json

```diff
@@ -47,9 +47,9 @@
     "postinstall:project": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
     "publish": "npm whoami && check-installed-dependencies && npm run compile && npm test && lerna publish",
     "postpublish": "npm run dependency-markdown && git add --all && git commit -m 'Updated DEPENDENCIES.md files'",
-    "start": "tt:serve --config webpack.config",
-    "start-heroku": "tt:serve-static --site build",
-    "start-static": "tt:serve-static --config webpack.config",
+    "start": "tt-serve --config webpack.config",
+    "start-heroku": "tt-serve-static --site build",
+    "start-static": "tt-serve-static --config webpack.config",
     "test": "npm run jest && npm run wdio && nightwatch",
```

Comment:
Do we need to add the `tds:generate-app-config` script like we have in terra-framework?

Reply:
Maw. Matt had it in there for testing purposes but when we run webpack or webdriverio, the app config is generated on-demand to a viral file system. Since we never want to check it in, this script is not relevant.

Reply:
yeah, we could go ahead and remove it from terra-framework